.github/workflows/gen-upgrade-on-manual-input.yml

```diff
@@ -0,0 +1,60 @@
+# When user trigger this workflow with custom version input, this action will do 3 things:
+# 1) it will create a directory with an empty upgrade handler in app/upgrades folder
+# 2) will increase an E2E_UPGRADE_VERSION variable in Makefile
+# 3) create a pull request with these changes to main
+
+name: On Trigger Gen Upgrade Handler
+
+on:
+  release: 
+    types: [published]
+
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Major vesrion to generate. Default value (v1) will ignore the action. Example format: v13'
+        required: true
+        default: 'v1'
+
+jobs:
+  post_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Specify input
+        run: |
+          env=${{ inputs.version }}
+          if [[ $env != '' ]]; then 
+            echo $env
+            echo "input=$env" >> $GITHUB_ENV
+          else
+            version=${{ github.event.release.tag_name }}
+            version=${version[@]:1}
+            version=(${version//./ })
+            version=v${version[0]}
+            echo $version
+            echo "input=$version" >> $GITHUB_ENV
+          fi
+
+      - name: Run version script
+        run: bash ./scripts/check_release.sh ${{ env.input }}
+
+      - name: Run post release script
+        if: env.MAJOR == 1    # 1 means vX of existing upgrade handler is smaller than A in tag vA.B.C
+        run: bash ./scripts/empty_upgrade_handler_gen.sh ${{ env.input }}
+
+      - name: Create PR
+        if: env.MAJOR == 1
+        uses: peter-evans/create-pull-request@v4
+        with:
```

Comment:
Can we name the to-be-created PR as "auto: code-gen upgrade handler v< version we are code-gening >"

Example here: https://github.com/osmosis-labs/osmosis/blob/a54797da63fc96059614754b4af65b424cebbaf3/.github/workflows/import_paths.yml#L66

Reply:
This is the new title of the pull request as seen here: https://github.com/MasterPi-2124/osmosis/pull/12. You can check the action's detail log here: https://github.com/MasterPi-2124/osmosis/actions/runs/3610164344

seems that the workflows still working well in both use cases and I am happy to get it merged!